spec/core/adjust.fmf

```diff
@@ -0,0 +1,62 @@
+summary: Adjust metadata based on the given context
+
+story:
+    As a user I want to adjust test, plan or story metadata
+    based on the given context such as product, distribution,
+    architecture or trigger.
+
+description: |
+    The ``adjust`` attribute allows to modify arbitrary object
+    metadata based on the context in which they are to be used.
+    For example tests may be relevant only for given environment,
+    the set of required packages may differ across distributions
+    or there might be a different set of plans executed for pull
+    requests versus for the full compose integration testing.
+
+    The value can be a dictionary or a list of dictionaries which
+    represent individual rules to be applied. Each rule contains
+    metadata to be merged into the original object. The following
+    three keys are reserved for rule evaluation:
+
+    when
+        The condition to be evaluated in order to decide if the
+        metadata should be merged. In the expression, you can use
+        any defined context :ref:`/spec/context/dimension`. See
+        the full `condition syntax`__ for details about supported
+        operators. This is a **required** key.
+
+    continue
+        By default, the first successful rule finishes the
+        evaluation and the rest is ignored. Set this attribute to
+        ``true`` to evaluate the following rule as well.
```

Comment:
where was the discussion about having this `false` by default instead?

Reply:
I believe it would be more user friendly to have it vice versa default

Reply:
The discussion was in #101. Why do you think `true` would be a better default for `continue`?

Reply:
Seems more natural to me, as a person who does not use this by default. if I write more rules, I expect all to be applied .... if I do not want them to be applied all, I would go to docs and look if there is a solution.

Now if I write two rules, I will be puzzled why the second one is not applied :( and it won't be clear. If there is some verbose message about this fact, then maybe it is acceptable. Still though feels like less prone to errors is to have it `True` by default

Reply:
The default value for `continue` has been changed to `True`. Documentation has been updated accordingly. Implementation is covered by https://github.com/psss/fmf/pull/92.